modules/backend/formwidgets/codeeditor/assets/less/codeeditor.less

```
@@ -119,4 +120,6 @@
         }
     }
 
-}
\ No newline at end of file
+}
+
+#cms-master-tabs { .field-codeeditor { .editor-toolbar { >ul { >li { >a { color: @color_3; } } } } } }
```

@ayumihamsaki I think @LukeTowers meant something like that:
```suggestion
#cms-master-tabs .field-codeeditor .editor-toolbar > ul  > li  > a { 
    color: @color_3; 
} 
```

---

I have to laugh! What was I thinking!